tests/cloud-init

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eux
+
+# Install LXD
+install_lxd
+
+# Configure LXD
+lxd init --auto
+
+IMAGE="${TEST_IMG:-ubuntu-minimal-daily:24.04}"
+
+echo "==> Create key pairs for tests"
+ssh-keygen -t ed25519 -f "./profile-key" -N ""
+ssh-keygen -t ed25519 -f "./additional-key" -N ""
+
+echo "==> Create configure profile for tests"
+lxc profile set default cloud-init.user-data "$(cat <<EOF
+#cloud-config
+users:
+  - name: root
+    ssh_authorized_keys: $(cat ./profile-key.pub)
+EOF
+)"
+
+echo "==> Create test instance"
+lxc launch "${IMAGE}" c -c cloud-init.ssh-keys.mykey="root:$(cat ./additional-key)"
+C_IPV4="$(lxc list c -c4 --format=csv | cut -d' ' -f1)"
+
```

Comment:
I believe we'd need `waitInstanceBooted` here to let `cloud-init` finish its job.